src/app/Sidebar.js

```diff
@@ -160,14 +166,22 @@ export default class Sidebar extends Component {
   };
 
   render() {
+    const { search, price, props, menu } = this.state;
     const user = this.props.auth.user;
-    const search = this.state.search;
 
-    if (_.has(this.state.feedPrice, 'base')) {
-      var power = formatter.vestToSteem(user.vesting_shares, this.state.props.total_vesting_shares, this.state.props.total_vesting_fund_steem);
-      var base = (this.state.feedPrice.base).replace(' SBD', '').replace(',', '');
-      var dollar = (parseFloat(base) * (parseFloat(user.balance) + parseFloat(power))) + parseFloat(user.sbd_balance);
-    }
+    const power = props
```

Comment:
(props) this.state.props is confusing would be nice to name as categoriesProp(if it is).